docs/root/start/quick-start/run-envoy.rst

```diff
@@ -104,6 +104,51 @@ The ``-c`` or ``--config-path`` flag tells Envoy the path to its initial configu
                    -c /envoy-custom.yaml
          ...
 
+   .. tab:: Windows Service
+
+      .. note::
+
+         This feature is still in Experimental state.
+
+      You can start Envoy as Windows Service that is managed under `Windows Service Control Manager <https://docs.microsoft.com/en-us/windows/win32/services/using-services/>`_. 
+
+      First, you need to create the service. Assuming you have a custom configuration in the current directory named ``envoy-custom.yaml``. After you create the service you
+      can start it.
+      
+      From an **administrator** prompt run the following commands (note that you need replace C:\EnvoyProxy\ with the path to the envoy.exe binary):
+
+      .. substitution-code-block:: console
+
+         > sc create EnvoyProxy binpath="C:\EnvoyProxy\envoy.exe --config-path envoy-demo.yaml"
+            [SC] CreateService SUCCESS
+         > sc start EnvoyProxy
+            SERVICE_NAME: envoyproxy
+               TYPE               : 10  WIN32_OWN_PROCESS
+               STATE              : 2  START_PENDING
+                                       (NOT_STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
+               WIN32_EXIT_CODE    : 0  (0x0)
+               SERVICE_EXIT_CODE  : 0  (0x0)
+               CHECKPOINT         : 0x0
+               WAIT_HINT          : 0x7d0
+               PID                : 3924
+               FLAGS              :
+         > sc query EnvoyProxy
+            SERVICE_NAME: envoyproxy
+               TYPE               : 10  WIN32_OWN_PROCESS
+               STATE              : 4  RUNNING
+                                       (STOPPABLE, PAUSABLE, ACCEPTS_SHUTDOWN)
```

Comment:
This is odd, current dwControlsAccepted omits pause-continue and includes preshutdown.

I'm guessing this example doesn't map to the current iteration of the patch? Could you refresh for consistency?

Reply:
The reason for that, I think is because the service is `Start Pending` and not `Started`

Reply:
I'm pointing out the second query, which claimed pausable

Reply:
thanks, for pointing out, there was a bug hiding in my code and this exposed it. fixed in https://github.com/envoyproxy/envoy/pull/14352/commits/44ce57b0a84e1c80792ff756a5cf530a930eb15d